portal-bridge/src/api/execution.rs

```diff
@@ -48,7 +49,15 @@ pub struct ExecutionApi {
 }
 
 impl ExecutionApi {
-    pub async fn new(provider: Provider, mode: BridgeMode) -> Result<Self, surf::Error> {
+    pub async fn new(
+        provider: Provider,
+        mode: BridgeMode,
+        requests_allowed_per_day: f64,
+    ) -> Result<Self, surf::Error> {
+        // Limits requests to no more then requests_allowed_per_day / 86400.0 per a second
```

Comment:
This comment is just a bit confusing...
*typo* `then` -> `than`
... 
Is it no more than `requests_allowed_per_day` or `86400.0 / second`?
Or is it no more than `(requests_allowed_per_day / 86400.0)` / `second`?

Imo changing `requests_allowed_per_day` to something like `daily_request_limit` helps with the understanding. 
It keeps it simple (in the comments and the cli help string), with just a "daily limit", that's the only variable users need to know about. Users can do the conversion on their own if they need to know the "per second" rate. I'd also do something like..
```rust
const SECONDS_IN_A_DAY: f64 = 86400.0;

let period = Duration::from_secs_f64(daily_request_limit / SECONDS_IN_A_DAY);
```